jaxrs/src/main/java/org/jboss/as/jaxrs/JaxrsParamHandler.java

```diff
@@ -105,10 +105,18 @@ private void updateServerConfig(OperationContext context, String attributeName,
             config.setResteasyJndiResources(value);
         } else if (JaxrsConstants.RESTEASY_LANGUAGE_MAPPINGS.equals(attributeName)) {
             config.setResteasyLanguageMappings(value);
+        } else if (JaxrsConstants.RESTEASY_MATCH_CACHE_ENABLED.equals(attributeName)) {
+            config.setResteasyMatchCacheEnabled(value);
+        } else if (JaxrsConstants.RESTEASY_MATCH_CACHE_SIZE.equals(attributeName)) {
+            config.setResteasyMatchCacheSize(value);
         } else if (JaxrsConstants.RESTEASY_MEDIA_TYPE_MAPPINGS.equals(attributeName)) {
             config.setResteasyMediaTypeMappings(value);
         } else if (JaxrsConstants.RESTEASY_MEDIA_TYPE_PARAM_MAPPING.equals(attributeName)) {
             config.setResteasyMediaTypeParamMapping(value);
+        } else if (JaxrsConstants.RESTEASY_PATCH_FILTER_DISABLED.equals(attributeName)) {
+            config.setResteasyPatchfilterDisabled(value);
+        } else if (JaxrsConstants.RESTEASY_PATCH_FILTER_LEGACY.equals(attributeName)) {
+            config.setResteasyPatchfilterLegacy(value);
         } else if (JaxrsConstants.RESTEASY_PREFER_JACKSON_OVER_JSONB.equals(attributeName)) {
```

Comment:
why not use switch on enum instead of the long series of string if-else comparison? The enum should be much faster.

Reply:
Hi @chengfang,

Good question. Like
```
       switch (attributeName) {
            case JaxrsElement.JAXRS_2_0_REQUEST_MATCHING.getLocalName():
                
        }
```
or even
```
        switch (attributeName) {
            case JaxrsConstants.JAXRS_2_0_REQUEST_MATCHING:
                ...
        }
```
Ok, I can change that.

-Ron

Reply:
I guess there are already Enum defined for every xml element and attribute names, so we can switch on enum instead of string.

Reply:
If you prefer. Wouldn't it be faster to switch on Strings instead of calling getLocalName()?

Reply:
For the `attributeName` passed to this method, we can get the corresponding enum value. I guess it's already defined in `JaxrsElement` or similar enum types.  Then we can switch on the enum value of the attribute.

```java
switch(attributeNameEnum) {
  case JaxrsElement.JAXRS_2_0_REQUEST_MATCHING:
```

Will something like that work in this case?

Reply:
Oh, I see. Looks good to me.

Reply:
Done